modules/menu/menu.c

```diff
@@ -793,6 +793,11 @@ static void ua_event_handler(struct ua *ua, enum ua_event ev,
 		menu_selcall(call);
 		break;
 
+	case UA_EVENT_CALL_REDIRECT:
+		info("menu: redirecting call to %s\n", prm + 4);
+		ua_connect(ua, NULL, NULL, prm + 4, VIDMODE_ON);
```

Comment:
`prm + 4` should be rewritten with proper parsing

Reply:
I was too lazy to do that and felt that parsing is not necessary, since we know how the parameter is formatted there and we know that the response code must have exactly three digits.  But can add parsing when it is sure that the PR is otherwise ready to go.

Reply:
And should we send here `UA_EVENT_CALL_BLIND_TRANSFER`?